rescuer.go

```diff
@@ -25,15 +25,13 @@ const (
 
 // Test-only properties.
 type rescuerTestSignals struct {
-	DiscardedJobs rivercommon.TestSignal[struct{}] // notifies when runOnce has discarded jobs from the batch
-	FetchedBatch  rivercommon.TestSignal[struct{}] // notifies when runOnce has fetched a batch of jobs
-	RetriedJobs   rivercommon.TestSignal[struct{}] // notifies when runOnce has retried jobs from the batch
+	FetchedBatch rivercommon.TestSignal[struct{}] // notifies when runOnce has fetched a batch of jobs
+	UpdatedBatch rivercommon.TestSignal[struct{}] // notifies when runOnce has updated rescued jobs from a batch
 }
 
 func (ts *rescuerTestSignals) Init() {
-	ts.DiscardedJobs.Init()
 	ts.FetchedBatch.Init()
-	ts.RetriedJobs.Init()
```

Comment:
Dropped down to only one signal because these are both part of the same operation now.